src/js/jsx/sections/style/ColorOverlayList.jsx

```diff
@@ -104,7 +105,11 @@ define(function (require, exports, module) {
                     this.props.index,
                     blendMode,
                     LayerEffect.COLOR_OVERLAY);
-            headlights.logEvent("edit", "color-overlay-blendmode-input", blendMode);
+
+            if (!this._logEventDebounced) {
+                this._logEventDebounced = synchronization.debounce(headlights.logEvent, null, 1000);
```

Comment:
It would be better to just make this a module-level variable instead of a member of the class.